model_compression_toolkit/core/common/quantization/quantization_config.py

```diff
@@ -46,86 +46,45 @@ class QuantizationErrorMethod(Enum):
     HMSE = 6
 
 
+@dataclass
 class QuantizationConfig:
+    """
```

Comment:
I know this is not new but it's written quite poorly English-wise, in case you'd like to revise it.